pkg/proxy/proxy.go

```diff
@@ -45,18 +49,29 @@ type EndpointRoute struct {
 type Config struct {
 	// DeviceName is the name of the device sent to Stripe to help identify the device
 	DeviceName string
-
 	// Key is the API key used to authenticate with Stripe
 	Key string
+	// URL to which requests are sent
+	APIBaseURL string
 
-	// EndpointsMap is a mapping of local webhook endpoint urls to the events they consume
+	// URL to which events are forwarded to
+	ForwardURL string
+	// Headers to inject when forwarding events
+	ForwardHeaders []string
+	// URL to which Connect events are forwarded to
+	ForwardConnectURL string
+	// Headers to inject when forwarding Connect events
+	ForwardConnectHeaders []string
+	// UseConfiguredWebhooks loads webhooks config from user's account
+	UseConfiguredWebhooks bool
```

Comment:
This is all specific to just webhooks, right? Is this shared with logs tailing?

Reply:
proxy is only for listen, so specific to webhooks. This has only be moved from the cobra cmd file to the proxy.go file